src/Main/WidgetFactory.cpp

```diff
@@ -26,11 +26,33 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include <QDialogButtonBox>
 #include <QLabel>
 #include <QMetaEnum>
+#include <iostream>
 
 using namespace FOEDAG;
 
 const QString DlgBtnBoxName{"SettingsDialogButtonBox"};
 
+// Local debug helpers
+#define WIDGET_FACTORY_DEBUG false
```

Comment:
why not qDebug() ?

Reply:
These are rare use debug prints that are only needed when actively developing in this area so i don't want the prints coming out everytime someone is running debug.